src/App.js

```diff
@@ -0,0 +1,56 @@
+import { Route, Routes } from "react-router";
+import { Layout } from "./Layout/Layout";
+import "./styles.css";
+import { Home } from "./components/Home/Home";
+import { Products } from "./components/Products/Products";
+import { Cart } from "./components/Cart/Cart";
+import { Wishlist } from "./components/Wishlist/Wishlist";
+import { ProductDetails } from "./components/Products/ProductDetails";
+import { Login } from "./components/Login/Login";
+import { Register } from "./components/Register/Register";
```

Comment:
You can compile all the exports from component folder inside a single file index.js in component folder.

Reply:
will do that!